tests/datadriven/test_reads.py

```diff
@@ -155,9 +155,6 @@ def testSampleIdEtc(self):
             readGroupInfo = self._readGroupInfos[readGroup.getLocalId()]
             gaReadGroup = readGroup.toProtocolElement()
             self.assertEqual(
-                readGroupInfo.sampleId,
```

Comment:
Added a test that reconstructs the bioSampleId and checks to make sure it is correct.